src/ViewInstance.js

```diff
@@ -518,6 +519,8 @@ class ViewInstance extends React.Component {
       );
     }
 
+    const instanceSubjects = get(instance, 'subjects', '').map(item => { return { value: item } });
```

Comment:
There is unnecessary return, use brackets around the object.

Reply:
Thanks